inception/inception-experimental-editor/src/main/java/de/tudarmstadt/ukp/inception/experimental/editor/basic/ExperimentalAnnotationEditor.java

```diff
@@ -55,7 +55,9 @@ public void renderHead(IHeaderResponse aResponse)
         super.renderHead(aResponse);
         aResponse.render(JavaScriptHeaderItem
                 .forScript("; localStorage.setItem('url','" + constructEndpointUrl() + "')", "0"));
```

Comment:
This bothered me quiet a while, because I did not find a better way to get the data to the client before the websocket connection. However, out of nowhere I found a simple solution now.

Reply:
Well, you can pass the URL as well as the HTML element to which the annotator should be bound as constructor arguments.